scripts/build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
+if [ -z $CONFIGURATION ]; then
+    CONFIGURATION="debug"
+fi
+
 export GRIMOIRE_ISTESTING=false
-swift build
+swift build --disable-sandbox -c $CONFIGURATION
```

Comment:
Could you explain why we are disabling the sandbox (for my personal education 😄)

Reply:
Because it fails without it and I'm not (yet) sure why 😁